### PR TITLE
Create persistent custom command script for startup

### PR DESCRIPTION
**Pull request by grhangelone**
As proposed in issue mattrayner#112

For the user to be able to create custom commands at container startup there should be a persistend script or folder, where we can add our commands like cron startup and such

As the script resides in /run, users can easily bind mount this directory to persistent storage
If the folder or script doesn't exist, we do nothing

### CODE REVIEW

File: supporting_files/run.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+c#!/bin/bash
```
Comment (pzhlkj6612) [inline supporting_files/run.sh:1]:
```suggestion
#!/bin/bash
```

Typo detected.

**Comment by pzhlkj6612** [comment]
Hi. if you want to make some experimental changes, I think you should switch to another branch in your forked repo, otherwise the update will always be sent to all subscribers.

I also note that you are working on the "master" branch. You may need to create a new branch and then change the default branch to it.

**Comment by grhangelone** [comment]
omg, sorry :)
I didn't mean to spam your inboxes.

**Comment by pzhlkj6612** [comment]
Don't worry. Happy coding :)

**Comment by grhangelone** [comment]
I have this working now with my own repo :)
For the new nextcould version I needed some PHP packages.

**Pull request closed without merging**